HW_7/logger.py

```diff
@@ -0,0 +1,12 @@
+from datetime import datetime
+
+
+def log(func):
+    def wrapper(*args):
+        with open('log.csv', 'a', encoding='utf-8') as log_file:
+            log_file.write(
+                f'{datetime.now()} – запущена функция {func.__name__} {"с аргументами" if args else "без аргументов"} ({func.__doc__})\n')
+
+        return func(*args)
+
+    return 
```

Comment:
Вот в чем дело. Этим декоратором вы декорируете функцию `start`. После декорации эта функция уже не будет функцией `start`. Она станет тем, что вернет функция  `log`, а функция лог у вас сейчас возвращает `None`, а этот объект не вызываемый, о чем вам и говорит ошибка при старте модуля main. Если вы переписываете с семинара, то делай те это до конца.
```suggestion
                f'{datetime.now()} – запущена функция {func.__name__} {f"с аргументами: (args)" if args else "без аргументов"} ({func.__doc__})\n')

        return func(*args)

    return wrapper
```